executor/executor_test.go

```diff
@@ -6508,3 +6508,223 @@ func (s *testSuite) TestTxnRetry(c *C) {
 	tk.MustExec("commit")
 	tk.MustQuery("select * from t").Check(testkit.Rows("10"))
 }
+
+func issue20975Prepare(c *C, store kv.Storage) (*testkit.TestKit, *testkit.TestKit) {
```

Comment:
It is better to `set tidb_enable_amend_pessimistic_txn = 0` in this function. The default value maybe change in future.